src/components/Login/ThirdPartyLoginComponents.jsx

```diff
@@ -90,4 +90,15 @@ function DexLoginButton({ handleClick }) {
   );
 }
 
-export { GithubLoginButton, GoogleLoginButton, GitlabLoginButton, DexLoginButton };
+function OIDCLoginButton({ handleClick, oidcName }) {
+  const classes = useStyles();
+  const loginWithName = oidcName || 'OIDC';
```

Comment:
Do you want to call this "OpenID Connect" instead of "OIDC"?
@raulkele ui appearance/real estate etc thoughts?

Reply:
I'd keep OIDC as it's the key used in the backend.
It's not like admin can override the value ;)

Reply:
In my opinion I'd say OIDC is good for an initial implementation. 'OIDC' is just the default value displayed in the UI and is customizable. If we get further feedback in the future, it can be changed then